components/form-builder/app/responses/Dialogs/ReportDialog.tsx

```diff
@@ -0,0 +1,263 @@
+import React, { useRef, useState } from "react";
```

Comment:
Copy+paste from DownloadTableDialog and removes anything related to the Confirm behavior